pkg/urbit/noun/manage.c

```diff
@@ -84,50 +84,30 @@ static sigjmp_buf u3_Signal;
 static uint8_t Sigstk[SIGSTKSZ];
 #endif
 
-#if 0
-/* _cm_punt(): crudely print trace.
```

Comment:
Do we have something like this for use in gdb?

Reply:
`u3m_p()`/`u3m_pretty()` usually work in gdb. I've never used this, but I'd be happy to leave it if it looks useful to you.

Reply:
If this works we should just uncomment it.  I often want to attach a debugger and see what it's working on in a hoon stack trace, but haven't been able to.

Reply:
I put it back, ifdef'd out, as a reminder to get trace printing working.